toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -606,8 +606,8 @@ zip-debuginfo-3.0-5.azl3.x86_64.rpm
 zlib-1.3-1.azl3.x86_64.rpm
 zlib-debuginfo-1.3-1.azl3.x86_64.rpm
 zlib-devel-1.3-1.azl3.x86_64.rpm
-zstd-1.5.5-2.azl3.x86_64.rpm
-zstd-debuginfo-1.5.5-2.azl3.x86_64.rpm
-zstd-devel-1.5.5-2.azl3.x86_64.rpm
-zstd-doc-1.5.5-2.azl3.x86_64.rpm
-zstd-libs-1.5.5-2.azl3.x86_64.rpm
+zstd-1.5.5-5.azl3.x86_64.rpm
+zstd-debuginfo-1.5.5-5.azl3.x86_64.rpm
+libzstd-1.5.5-5.azl3.x86_64.rpm
```

Comment:
the `toolchain_*.txt` files are listed in alphabetical order. Can you please alphabetize this file

Reply:
wait the toolchain_*.txt packages are alphabetized but the pkggen_*.txt packages are individually installed in order? that is confusing on many levels...

Reply:
ok i sorted the file correctly alphabetically, there was quite a bit out of order.